Makefile

```diff
@@ -47,6 +47,9 @@ clean:
 	rm -rf build/configuration-as-code/target
 	rm -rf build/essentials/target
 
+mrproper: clean
```

Comment:
I tend to use that naming since this is the name used in the kernel IIRC, and has somehow become a standard name when one really want to clean things up aggressively. But I won't fight for this one :)

Reply:
https://www.linuxquestions.org/questions/slackware-14/difference-between-make-mrproper-and-make-clean-473144/ for example :).
In our case, removing the `build` directory, and maybe more things in the future.

Reply:
I'm confused, why not just make that the `clean` target? Having a "clean" and "veryveryclean" target is silly IMHO

Reply:
I don't feel that strongly. It's just a similar rationale as for the Kernel I suppose: calling `make clean` is supposed to be enough 90% of the times, and is pretty fast. `make mrproper` would be cleaning up even things that you generally keep and shouldn't an issue, and will also be costly to rebuild. But I added this IIRC when ./build was still having locally built plugins, it possibly does not so much sense now.

Reply:
fixed. Removed.